tools/binary-compatibility/reference-public-api/reaktive.txt

```diff
@@ -1225,10 +1225,16 @@ public final class com/badoo/reaktive/subject/SubjectExtKt {
 	public static final fun isActive (Lcom/badoo/reaktive/subject/Subject;)Z
 }
 
-public abstract interface class com/badoo/reaktive/subject/behavior/BehaviorSubject : com/badoo/reaktive/subject/Subject {
+public abstract interface class com/badoo/reaktive/subject/behavior/BehaviorObservable : com/badoo/reaktive/observable/Observable {
 	public abstract fun getValue ()Ljava/lang/Object;
 }
 
+public abstract interface class com/badoo/reaktive/subject/behavior/BehaviorRelay : com/badoo/reaktive/subject/Relay, com/badoo/reaktive/subject/behavior/BehaviorObservable {
+}
+
+public abstract interface class com/badoo/reaktive/subject/behavior/BehaviorSubject : com/badoo/reaktive/subject/Subject, com/badoo/reaktive/subject/behavior/BehaviorRelay {
```

Comment:
I think it should be fine.
https://wiki.eclipse.org/Evolving_Java-based_APIs_2#Achieving_API_Binary_Compatibility
Move API method up type hierarchy | If method in supertype need not be reimplemented by Client | Binary compatible
-- | -- | --

Reply:
Do you like the change in general?

Reply:
Do not have any objections.